api/c/indigo/src/option_manager.h

```diff
@@ -19,6 +19,11 @@
 #ifndef __otion_manager_h__
 #define __otion_manager_h__
 
+#ifdef _MSC_VER
+#pragma warning(push)
+#pragma warning(disable : 4251)
```

Comment:
4251 and 4275 warnings in MSVC is about dllexport class derived from non-dllexport class or containing non-dllexport members.